profiling/plot/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3
+
+ADD requirements.txt /
+RUN pip install -r requirements.txt
```

Comment:
Is the requirements file checked in? Or, more generally, can you update the README to describe how to use this Dockerfile?

Does this even need a dockerfile? Consider embedding this in a script directly like:

```sh
docker -f- . <<EOF
FROM python:3
ADD ...
EOF
```

Reply:
oh, TIL you can do that, thanks!